action.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# This script is ran by the github actions to install fluvio in
+# GitHub Action Workflows.
+
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+        echo "Installing Fluvio Local Cluster"
+        
+        # Install Minikube for ubuntu
+        # Pre-Installation Check 
+        grep -E --color 'vmx|svm' /proc/cpuinfo
+
+        # Install
+        curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-${PLATFORM} \
+        && chmod +x minikube
+        sudo install minikube /usr/local/bin/
+
+        # Start Minikube with `none` driver
+        minikube start --driver=none
+
+        # Install Helm
+        curl -sSL https://get.helm.sh/helm-${HELM_VERSION}-${PLATFORM}.tar.gz | tar zx
```

Comment:
can GitHub Action script can refer to another script? since we already have helm install script?

Reply:
commit 72da370 now adds a step to install helm using the ci-replace-helm.sh script